packages/wallet/ui/src/AgoricLogo.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const AgoricLogo = () => {
+  const pathStyle = { fill: '#bb2d40' };
+
+  return (
+    <svg
```

Comment:
For some reason loading a static resource was messing with the ability of the bridge to connect to localstorage, so just include this in the bundle

Reply:
I recall there are some bad ways to use SVG in React but I don't remember whether this is one of them :)  Either way, seems fine for now. Optimize after correctness.

Reply:
Fixed in https://github.com/Agoric/agoric-sdk/pull/5984/commits/d823ccd6a70b6d94606a55cc381b07f4edb35c97, but I'll leave this in anyway because it's small enough and will look better not loading in separately.